bmi_config.cfg

```diff
@@ -37,16 +36,22 @@ rpc_server_port = <port of rpc server>
 
 # this section is for specifying tftp settings
 [tftp]
-pxelinux_url = <path to pxelinux.cfg>
-ipxe_url = <path to location of ipxe files>
+pxelinux_path = <path to pxelinux.cfg>
+ipxe_path = <path to location of ipxe files>
 
 # this section is for http config
+# Should think of a better name for this section
```

Comment:
Na too boring name. It is obvious that it is related to http config as the section name is http and it is a config file.
Need something that gives more clarity than http. If you will http is sufficient I am ok with it.